src/schema.cpp

```diff
@@ -194,6 +194,13 @@ const char* MODELS_CONFIG_SCHEMA = R"({
 				"params": {
 					"type": "object",
 					"additionalProperties": { "type": "string" } 
+				},
+				"demultiply_count": {
+					"type": "integer",
+					"minimum": 0
```

Comment:
Here the minimum should be 2 -> there is no sense to set 0 -> it will be deterministic for pipeline to process nothing. With 1 there is no need for demultiplication as well, so we should start with 2.